src/HealthGPS.Console/model_parser.cpp

```diff
@@ -59,10 +59,31 @@ hgps::RiskFactorSexAgeTable load_risk_factor_expected(const host::Configuration
     return table;
 }
 
+std::unique_ptr<hgps::DummyModelDefinition>
+load_dummy_model_definition(hgps::RiskFactorModelType type, const poco::json &opt) {
+    MEASURE_FUNCTION();
+
+    // Get dummy model parameters
+    std::vector<hgps::core::Identifier> names;
+    std::vector<double> values;
+    std::vector<double> policy;
+    for (const auto &[key, json_params] : opt["ModelParameters"].items()) {
+        names.emplace_back(key);
+        values.emplace_back(json_params["Value"].get<double>());
+        policy.emplace_back(json_params["Policy"].get<double>());
+    }
+
+    return std::make_unique<hgps::DummyModelDefinition>(type, std::move(names), std::move(values),
+                                                        std::move(policy));
+}
+
 std::unique_ptr<hgps::RiskFactorModelDefinition>
 load_static_risk_model_definition(const std::string &model_name, const poco::json &opt,
```

Comment:
And, as we are updating this function, let's take the opportunity to add a docstring :smiley:

Reply:
No preference at all - I don't know the common practices of C++. Having said that, the actual functionality is in the source code, not in the header, so I think I will find useful to have a description of the function alongside the code to fully understand what's going on.

Reply:
Like in the link you gave, have a 'developer' docstring in the .cpp and a 'user' docstring in the header? It's not a bad idea tbh. Bit of extra work, but I can see the use. Worth debating next meet.